npm/test/basics.js

```diff
@@ -781,8 +791,10 @@ test("debug service getting breakpoints after loaded source succeeds when file n
   const debugService = getDebugServiceWorker();
   try {
     const result = await debugService.loadSource(
-      "test.qs",
-      `namespace Sample {
+      [
```

Comment:
Did you add any tests for multiple files, or just update the single file tests for the new structure?